service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -602,7 +602,7 @@ public HearingBooking getMostUrgentHearingBookingAfter(LocalDateTime time) {
 
     public boolean hasFutureHearing(List<Element<HearingBooking>> hearingBookings) {
         return isNotEmpty(hearingBookings) && hearingBookings.stream()
-            .anyMatch(futureHearing -> futureHearing.getValue().startsAfterToday());
+            .anyMatch(hearing -> hearing.getValue().startsAfterToday());
```

Comment:
Any reason for this revert? I thought that the code smell was because the variable was called `hearing`?

Reply:
Although a better name might be hearingBooking